modules/govuk/manifests/node/s_search.pp

```diff
@@ -25,4 +25,14 @@
   if ! $::aws_migration {
     include govuk_elasticsearch::local_proxy
   }
+
+  # Set Plek for AWS to Carrenza communication
+  if ( ( $::aws_migration == 'search' ) and ($::aws_environment == 'staging') ) or ( ($::aws_migration == 'search' ) and ($::aws_environment == 'production') ) {
+    $app_domain = hiera('app_domain')
+
+    govuk_envvar {
+      'PLEK_SERVICE_EMAIL_ALERT_API_URI': value  => "https://email-alert-api.${app_domain}";
+      'PLEK_SERVICE_PUBLISHING_API_URI': value  => "https://publishing-api.${app_domain}";
+    }
+  }
```

Comment:
It might be clearer to put this configuration in the app, rather than on the node? As my understanding is that this directly relates to things accessed by search-api?

Reply:
There is prior art for this being configured on the node (https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk/manifests/node/s_frontend.pp#L32-L41).  `search-api` is the only app running on this node and I expect this code to be removed once the AWS migration is completed.

Reply:
Ok, fair enough.